hi-ml-histopathology/testhisto/testhisto/mocks/slides_generator.py

```diff
@@ -10,19 +11,26 @@
 from torch import Tensor
 from tifffile import TiffWriter
 
-from typing import Any, Tuple, List, Union
+from typing import Any, Optional, Tuple, List, Union
 from histopathology.datasets.panda_dataset import PandaDataset
 from testhisto.mocks.base_data_generator import MockHistoDataGenerator, MockHistoDataType
 
 
+class TilesPositioningType(Enum):
+    DIAGONAL = 0
+    RANDOM = 1
+
+
 class MockPandaSlidesGenerator(MockHistoDataGenerator):
     """Generator class to create mock WSI on the fly. A mock WSI resembles to:
                                 [**      ]
                                 [  **    ]
```

Comment:
I think the comment is no longer valid if "random" is used?

Reply:
I added this "if tiles positioning is diagonal. Tiles are positioned randomly on the WSI grid when tiles positioning type is random." as clarification. Do you think I should just remove the illustration? It's helpful to better understand the purpose of other parameters.